armi/cases/case.py

```diff
@@ -226,6 +226,10 @@ def getPotentialParentFromSettingValue(self, settingValue, filePattern):
         """
         Get a parent case based on a setting value and a pattern.
 
+        This is a convenient way for a plugin to express a dependency. It uses the
+        ``match.groupdict`` functionality to pull the directory and case name out of a
+        specific setting value an regular expression.
+        
```

Comment:
I am like 95% sure `black` is failing because there is extra whitespace on this line.

Reply:
Ah, you're right. Fixed!